### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by DanBotero**
i18n for text tag elements in the templates.

### CODE REVIEW

File: src/app/demo/templates/templates.component.html [hunk 1]
```diff
@@ -22,41 +22,44 @@
...
+    <h1 i18n>Templates / Exercise</h1>
+    <h2 i18n>In the next slide you'll edit component template to create a simple header and search form. Ther result will look like
       this:</h2>
 
     <slides-browser-window>
       <div>
-        <h1>MewTube</h1><input placeholder="video" type="text">
+        <h1 i18n>MewTube</h1>
+        <input placeholder="video" type="text">
         <button>Search</button>
```
Comment (tessaSAC) [inline src/app/demo/templates/templates.component.html]: Do you need to put `i18n` here too?

Comment (tessaSAC) [inline src/app/demo/templates/templates.component.html]: (on search button text)

File: src/app/demo/typescript/typescript.component.html [hunk 2]
```diff
@@ -85,11 +85,11 @@
 
   <!--  NUMBER-TYPE  -->
   <slides-slide slidesIntrojsDirective id="type-mini-exercise" milestone="TypeScript">
-    <h1 data-step="1" data-intro="Welcome to the first coding exercise!">TypeScript / Type System</h1>
-    <h2>2 + 2 = 4? Not always the case in JavaScript. Luckily TypeScript allows us to add types and detect the issue
+    <h1 data-step="1" data-intro="Welcome to the first coding exercise!" i18n>TypeScript / Type System</h1>
+    <h2 i18n>2 + 2 = 4? Not always the case in JavaScript. Luckily TypeScript allows us to add types and detect the issue
       early</h2>
     <div data-step="2" data-intro="These are the exercise instructions." class="exercise"
```
Comment (tessaSAC) [inline src/app/demo/typescript/typescript.component.html]: Also do you need them on the tour steps?

Comment (tessaSAC) [inline src/app/demo/typescript/typescript.component.html]: (`data-intro`)

File: src/app/demo/templates/templates.component.html (hunk 1 above)
Comment (kirjs) [inline src/app/demo/templates/templates.component.html]: This section will go away

File: src/app/demo/typescript/typescript.component.html (hunk 2 above)
Comment (kirjs) [inline src/app/demo/typescript/typescript.component.html]: Yes, we do, added an issue

File: src/app/demo/templates/templates.component.html (hunk 1 above)
Comment (tessaSAC) [inline src/app/demo/templates/templates.component.html]: :sob: rip

**Pull request merged**